garmin_fit.cc

```diff
@@ -210,6 +210,21 @@ GarminFitFormat::fit_getuint32()
   }
 }
 
+QString
+GarminFitFormat::fit_getstring(int size)
+{
+  if (fit_data.len < size) {
+      throw ReaderException("record truncated: expecting " + std::to_string(size) + " bytes, but only got " + std::to_string(fit_data.len) + ".");
```

Comment:
please correct the indentation here.
Also please delete the trailing white space on two other lines.

This can be done manually or with "astyle --options=astylerc garmin_fit.cc"